src/NLog.Web.AspNetCore/LayoutRenderers/AspNetRequestStreamIdLayoutRenderer.cs

```diff
@@ -0,0 +1,28 @@
+﻿#if NET5_0_OR_GREATER
+using System.Text;
+using Microsoft.AspNetCore.Connections.Features;
+using NLog.LayoutRenderers;
+using NLog.Web.Internal;
+
+namespace NLog.Web.LayoutRenderers
+{
+    /// <summary>
+    /// Represents the long int identifier for the stream.
+    /// Uses IStreamIdFeature
+    /// </summary>
+    /// <remarks>
+    /// <code>${aspnet-request-stream-id}</code>
+    /// </remarks>
+    [LayoutRenderer("aspnet-request-stream-id")]
+    public class AspNetRequestStreamIdLayoutRenderer : AspNetLayoutRendererBase
```

Comment:
This can show when the long int of the socket stays the same or changes, so you can see when connection is reused, or closed and re-opened.  Similar to log thread ID or similar.

Reply:
Can you update the XML docs ?

Reply:
Done as requested